src/cmd/bootstrap.ts

```diff
@@ -125,6 +127,10 @@ export const bootstrapValues = async (): Promise<void> => {
   if (!existsSync(`${env.ENV_DIR}/env`)) {
     debug.log(`Copying basic values`)
     await copy(`${cwd}/.values/env`, `${env.ENV_DIR}/env`, { overwrite: false, recursive: true })
+
+    // Generate passwords and merge with values only if there are no values already
+    const generatedSecrets = yaml.load(await generateSecrets())
+    await mergeValues(generatedSecrets)
```

Comment:
omg, you _are_ overwriting secrets with freshly generated secrets, even after I _specifically and repeatedly_ told you how to avoid this....disappointing